go-controller/cmd/ovnkube-identity/ovnkubeidentity.go

```diff
@@ -0,0 +1,377 @@
+					errorList = append(errorList, err)
+				}
+			}()
+		}
+
+		if !cliCfg.disableApprover {
+			startWg.Add(1)
+			go func() {
+				defer startWg.Done()
+				if err := runCSRApproverManager(c.Context, c.App.Name, restCfg); err != nil {
+					errorList = append(errorList, err)
+				}
+			}()
+		}
+
+		return errors.NewAggregate(errorList)
+	}
+
+	c.Flags = []cli.Flag{
+		&cli.StringFlag{
+			Name:        "kubeconfig",
+			Usage:       "kubeconfig path",
+			Destination: &cliCfg.kubeconfig,
+		},
+		&cli.StringFlag{
+			Name:        "k8s-apiserver",
+			Usage:       "URL of the Kubernetes API server (not required if --kubeconfig is given)",
+			Destination: &cliCfg.apiServer,
+		},
+		&cli.StringFlag{
+			Name:        "lease-namespace",
+			Usage:       "namespace in which the leader election lease object will be created (pod namespace by default)",
+			Destination: &cliCfg.leaseNamespace,
+		},
+		&cli.IntFlag{
+			Name:        "loglevel",
+			Usage:       "log level. Note: info, warn, fatal, error are always printed no matter the log level. Use 5 for debug (default: 4)",
+			Destination: &cliCfg.logLevel,
+			Value:       4,
+		},
+		&cli.BoolFlag{
+			Name:        "disable-webhook",
+			Usage:       "If set webhook will not be started, cannot be used with disable-approver",
+			Destination: &cliCfg.disableWebhook,
+			Value:       false,
+		},
+		&cli.BoolFlag{
+			Name:        "disable-approver",
+			Usage:       "If set CSR approver will not be started, cannot be used with disable-webhook",
+			Destination: &cliCfg.disableApprover,
+			Value:       false,
+		},
+		&cli.StringFlag{
+			Name:        "webhook-cert-dir",
+			Usage:       "directory that contains the server key and certificate",
+			Destination: &cliCfg.certDir,
+		},
+		&cli.StringFlag{
+			Name:        "webhook-host",
+			Usage:       "the address that the webhook server will listen on",
+			Value:       "localhost",
+			Destination: &cliCfg.host,
+		},
+		&cli.IntFlag{
+			Name:        "webhook-port",
+			Usage:       "port number that the webhook server will serve",
+			Value:       webhook.DefaultPort,
+			Destination: &cliCfg.port,
+		},
+		&cli.StringFlag{
+			Name:        "metrics-address",
+			Usage:       "address that the metrics server will serve. (Default: \"0\", which means disabled)",
+			Value:       "0",
+			Destination: &cliCfg.metricsAddress,
+		},
+		&cli.BoolFlag{
+			Name:        "enable-interconnect",
+			Usage:       "Configure to enable ovn interconnect checks",
+			Destination: &cliCfg.enableInterconnect,
+			Value:       false,
+		},
+		&cli.BoolFlag{
+			Name:        "enable-hybrid-overlay",
+			Usage:       "Configure to enable hybrid overlay checks",
+			Destination: &cliCfg.enableHybridOverlay,
+			Value:       false,
+		},
+		&cli.StringSliceFlag{
+			Name:        "extra-allowed-user",
+			Usage:       "Configure extra user that is allowed to modify annotations protected by the webhook, can be used multiple times",
```

Comment:
nit, add who is currently using this in the usage?

Reply:
I would like to avoid adding the users here as this might change in the future.